ui/helpers/utils/export-utils.js

```diff
@@ -1,19 +1,23 @@
 import { getRandomFileName } from './util';
 
+function encodeB64(text, cb) {
+  const buf = new TextEncoder().encode(text);
+  // eslint-disable-next-line no-undef
+  const reader = new FileReader();
+  reader.onload = () => cb(reader.result.split(',')[1]);
+  // eslint-disable-next-line no-undef
+  reader.readAsDataURL(new Blob([buf]));
+}
+
 export function exportAsFile(filename, data, type = 'text/csv') {
-  // eslint-disable-next-line no-param-reassign
-  filename = filename || getRandomFileName();
-  // source: https://stackoverflow.com/a/33542499 by Ludovic Feltz
-  const blob = new window.Blob([data], { type });
-  if (window.navigator.msSaveOrOpenBlob) {
-    window.navigator.msSaveBlob(blob, filename);
-  } else {
+  encodeB64(data, (b64) => {
+    // eslint-disable-next-line no-param-reassign
+    filename = filename || getRandomFileName();
     const elem = window.document.createElement('a');
-    elem.target = '_blank';
-    elem.href = window.URL.createObjectURL(blob);
+    elem.href = `data:${type};Base64,${b64}`;
```

Comment:
can we just use `btoa`? https://developer.mozilla.org/en-US/docs/Web/API/btoa

Reply:
unfortunately `btoa` fails on special chars that we have in the exported json files that `TextEncoder` successfully deals with (I learned the hard way)

Reply:
interesting -- what about `Buffer.from(xyz, 'utf8').toString('base64')`

Reply:
e95a668f0